other/other.go

```diff
@@ -0,0 +1,17 @@
+package other
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func RespondWithError(w http.ResponseWriter, code int, message string) {
```

Comment:
Are these function still needed given that you switched to Gin? At least I couldn't find usages of it in this PR. Correct me if I'm wrong.